Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -388,48 +410,13 @@ extension SearchViewController: UITableViewDataSource {
         case .SearchSuggestions:
             return searchEngines.shouldShowSearchSuggestions ? 1 : 0
         case .BookmarksAndHistory:
-            return data.count
+            return data?.count ?? 0
```

Comment:
Yeah, commit c8f2916 changed this. At one point, I think I had the Loader returning nil. But I don't think this commit is necessary anymore, so I'll drop it.